WebApplication/ClientApp/src/components/models/Category.tsx

```diff
@@ -0,0 +1,13 @@
+
+export class Category {
+    id: number;
+    title: string;
+    dedicatedamount: number;
+
+    constructor(id: number, title: string, dedicatedamount: number) {
+        this.id = id;
+        this.title = title;
+        this.dedicatedamount = dedicatedamount;
+    }
+
+}
```

Comment:
Maybe a class is not the best option if we are only displaying data, consider using type instead.